src/run.jl

```diff
@@ -543,5 +543,48 @@ function compute_linear_structure(parameters_file=nothing, source_terms=nothing;
 end
 
 
+"""
+    compute_jacobian_dg(parameters_file=nothing; verbose=false, refinement_level_increment=0, parameters...)
+
+Uses DG right hand side operator and simple second order finite difference to compute the Jacobian of the operator.
+The linearisation state is the initial condition from the parameter file.
+"""
+function compute_jacobian_dg(parameters_file=nothing; verbose=false, refinement_level_increment=0, parameters...)
```

Comment:
I think `refinement_level_increment` should be omitted unless you really want this feature, as IMHO it only makes sense for convergence tests.

Reply:
Same for `compute_linear_structure`

Reply:
changed now